core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -1376,6 +1379,42 @@ public int getFileId(TachyonURI path) throws InvalidPathException {
     }
     return ret;
   }
+  
+  /**
+   * @return the total bytes on each storage tier.
+   */
+  public List<Long> getTotalBytesOnTiers() {
+    List<Long> ret = new ArrayList<Long>();
```

Comment:
Oh I see, the arraylist values defaulting to zero makes it easier to write the for loop.
I meant to instantiate the ArrayList like 

```java
List<Long> ret = new ArrayList<Long>(StorageLevelAlias.values().length);
```

It would just be a minor optimization to avoid resizing the arraylist if there happen to be more than 10 storage levels.